OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/immutability/ClassImmutabilityAnalysis.scala

```diff
@@ -1,22 +1,28 @@
 /* BSD 2-Clause License - see OPAL/LICENSE for details. */
 package org.opalj
-package br
+package tac
```

Comment:
Why were this and the `TypeImmutabilityAnalysis` moved to `tac`? I don't see them using the TAC, so they should have remained in `br`.